qa/tasks/cephfs/fuse_mount.py

```diff
@@ -138,7 +142,15 @@ def _add_valgrind_args(self, mount_cmd):
 
         return mount_cmd
 
-    def _list_fuse_conns(self):
+    def _list_fuse_conns_cmd(self):
```

Comment:
This change needs a review from CephFS.

CC @batrick, @vshankar.

Reply:
This is a refactoring that is meant to be fully backward compatible. The reason for it is to inject macOs custom commands where appropriate without having to change the common logic.